lsp-rust.el

```diff
@@ -666,38 +666,48 @@ The command should include `--message=format=json` or similar option."
     (display-buffer buf)))
 
 ;; runnables
-(defvar lsp-rust-analyzer--last-runnable nil)
+(defvar-local lsp-rust-analyzer--last-runnable-label nil)
```

Comment:
Not sure whether making this buffer-local is a good idea -- it's nice to be able to run a test and then rerun it while working on the main code

Reply:
That's a good point I haven't thought about. In that case I probably shouldn't change the design here, i.e. it should store the full `Runnable` record rather than just a label.

Reply:
By the way if we use `compile` instead of `compilation-start` (I am not sure if this is possible) then one could use `recompile` to run the last command.

Reply:
Had a quick look at `compilation-start` and `compile`. I guess `compilation-start` was used because it needs to customize the buffer name and mode?